front-end/src/components/SectionCard/index.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+
+import styles from './SectionCard.module.css'
+import { Icon } from '../Icon'
+
+export const SectionCard = () => {
+  return (
+    <div className={styles.container}>
+      <h1 className={styles.title}>
+        This is a Title
+        <p className={styles.username}>User Name</p>
+      </h1>
+      <div
+        onClick={() => console.log('Clicked Edit button')}
```

Comment:
Awesome work ! I guess you could add the onClick to the Icon as well so you don't have to wrap it with another div that uses the onClick. You could even add the `className` as well^^